src/tools/compiletest/src/header.rs

```diff
@@ -672,7 +673,8 @@ pub fn line_directive<'line>(
 /// This is generated by collecting directives from ui tests and then extracting their directive
 /// names. This is **not** an exhaustive list of all possible directives. Instead, this is a
 /// best-effort approximation for diagnostics.
-const DIAGNOSTICS_DIRECTIVE_NAMES: &[&str] = &[
+// tidy-alphabetical-start
+const KNOWN_DIRECTIVE_NAMES: &[&str] = &[
```

Comment:
Shouldn't we put `tidy-alphabetical-start` inside the list?

Reply:
Oops, yeah that would make more sense...

Reply:
Fixed.